packages/lit-dev-content/site/docs/v2/templates/directives.md

```diff
@@ -1831,7 +1831,7 @@ import {asyncAppend} from 'lit/directives/async-append.js';
 <td class="wide-cell">
 
 ```ts
-asyncAppend(iterable: AsyncIterable)
+asyncAppend(iterable: AsyncIterable, mapper?: Function)
```

Comment:
I actually like this to be verbose so it shows the index is also provided.
How about doing it multi-line to make it fit?
```suggestion
asyncAppend(
  iterable: AsyncIterable,
  mapper?: (item: Item, index?: number) => unknown
)
```

Reply:
Done! Great suggestion!